src/include/planner/operator/logical_operator.h

```diff
@@ -135,7 +135,14 @@ class KUZU_API LogicalOperator {
 
 protected:
     void createEmptySchema() { schema = std::make_unique<Schema>(); }
-    void copyChildSchema(uint32_t idx) { schema = children[idx]->getSchema()->copy(); }
+    void copyChildSchema(uint32_t idx) { 
```

Comment:
Let's limit the change to `LogicalExplain` only and keep LogicalOperator as it is